xwiki-platform-core/xwiki-platform-refactoring/xwiki-platform-refactoring-api/src/test/java/org/xwiki/refactoring/internal/listener/BackLinkUpdaterListenerTest.java

```diff
@@ -19,17 +19,29 @@
  */
 package org.xwiki.refactoring.internal.listener;
 
+import static org.junit.jupiter.api.Assertions.assertEquals;
```

Comment:
Unless I've something miss-configured in my IDE, I find it surprising to see those imports reordered.

Reply:
I missed that the imports were rearranges when adding the changes for this file. I'll change them back
But probably it's something from my IDE since I get this quite often

Reply:
This should help: https://dev.xwiki.org/xwiki/bin/view/Community/CodeStyle/JavaCodeStyle/#HConfiguringyourIDEtousetheXWikicodestyle
If it does not let me know :)

Reply:
Found the problem. My shortcut for organizing imports (from eclipse) moved it, so it's not related to code style